internal/aws/metrics/metric_calculator.go

```diff
@@ -80,10 +80,12 @@ type Key struct {
 }
 
 func NewKey(metricMetadata interface{}, labels map[string]string) Key {
-	var kvs []attribute.KeyValue
+	kvs := make([]attribute.KeyValue, len(labels))
```

Comment:
```suggestion
	kvs := make([]attribute.KeyValue, 0, len(labels))
```

Reply:
OK here I agree this can use the append style.

Reply:
addressed